create_test_data.py

```diff
@@ -0,0 +1,91 @@
+"""
+
+This is a script for generating test holdshelf events in the local psql db.
+
+Usage:
+    AWS_PROFILE=nypl-digital-dev ENVIRONMENT=development \
+        python create_test_data.py
+
+"""
+
+import os
+import datetime
+
+from nypl_py_utils import PostgreSQLClient, PostgreSQLClientError
+from nypl_py_utils.functions.config_helper import load_env_file
+
+HOLDS = [
+    [1, 10001, 'i', 'YYYY-MM-DDT08:11:01', 'mal'],
+    [2, 10002, 'i', 'YYYY-MM-DDT08:31:01', 'mal']
+]
+
+ITEMS = [
+    (10001, 101, 'maf92'),
+    (10002, 102, 'maf92')
+]
+
+sierra_client = None
+
+
+def connect():
+    global sierra_client
```

Comment:
Instead of using a global variable it might be cleaner to stick all these methods in a class kind of like you did with sierra_db_client.

Reply:
I made it into a class. Global did feel wrong.